src/test/java/com/teragrep/pth10/evalTest.java

```diff
@@ -2299,9 +2266,8 @@ public void parseMvdedupCatalystTest() {
             assertEquals("[1, 2, 3, 4]", lstA.get(0).getList(0).toString());
         });
     }
-    
+
     // Test eval method mvfilter(x)
-    // TODO
     @Disabled
```

Comment:
Perhaps add comment why this is disabled. If I remember correctly, `mvfilter` is not yet implemented.